README.md

```diff
@@ -90,6 +90,66 @@ WIP.
 
 ### Use of Swagger with Testing Framework
 
+## Testing with Sinopia editor
+
+After bringing up the server with or without docker, run the `loadRTs.sh` script in the root directory and this will create the following containers and groups in trellis:
+
+- a base repository:
+```
+{
+  "@id": "http://localhost:8080/",
+  "label": "Sinopia LDP Server",
+  "@context": {
+    "label": {
+      "@id": "http://www.w3.org/2000/01/rdf-schema#label"
+    }
+}
+```
+- a 'repository' repository:
+```
+{
+  "@id": "http://localhost:8080/repository",
+  "label": "repository",
+  "contains": "http://localhost:8080/repository/ld4p",
+  "@context": {
+    "contains": {
+      "@id": "http://www.w3.org/ns/ldp#contains",
+      "@type": "@id"
+    }
+}
+```
+- an 'ld4p' group repository that contains the json resource templates in the fixtures/profile_defs/bibframe folder:
+```
```

Comment:
I may have missed the bus ... but would we want to use the latest from https://github.com/LD4P/sinopia_sample_profiles/tree/master/profiles instead of the fixtures here??   

Is it not time?  If not, *when* will it be time?

Is it a matter of writing a script to pull out the individual resource templates from the profiles??  And using github raw to get the actual profiles?

Reply:
So, I do have some [preliminary code](https://github.com/LD4P/gh-resource-templates) that uses Github to retrieve the Sinopia Sample Profiles from master, still needs to parse out all of the Resource Templates and use the sinopia_client npm module to ingest into Trellis.

Reply:
I guess my feeling is ... we know we have to do this ... is there a compelling reason NOT to do it now?   And should it live here in this repo, or stay separate, like in gh_xxx repo Jeremy created?

I'm ok with "not now" if there's a reason for it.  ow., aren't we creating extra work for ourselves later?